src/sql/workbench/contrib/notebook/browser/cellViews/cellToolbar.component.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import 'vs/css!./cellToolbar';
+import { Component } from '@angular/core';
+import { localize } from 'vs/nls';
+
+const ButtonEdit = localize('buttonEdit', "Edit");
+const ButtonClose = localize('buttonClose', "Close");
+const ButtonAdd = localize('buttonAdd', "Add new cell");
+const ButtonMoveDown = localize('buttonMoveDown', "Move cell down");
+const ButtonMoveUp = localize('buttonMoveUp', "Move cell up");
+const ButtonDelete = localize('buttonDelete', "Delete cell");
+
+@Component({
+	selector: 'cell-toolbar-component',
+	template: `
+		<ul class="cell-toolbar">
```

Comment:
For consistency, would be great to have this defined in its own html file.

Reply:
Done. I did this for both cellToolbar and markdownToolbar components.

Reply:
Note: I'm leaving the cellToolbar commented out in both the textCell and codeCell components. These will in fact be used soon, so I think it's best to leave them there.